package.json

```diff
@@ -110,6 +127,7 @@
     "test:command-reference": "./bin/run commandreference:generate --erroronwarnings",
     "test:deprecation-policy": "./bin/run snapshot:compare",
     "test:nuts": "nyc mocha \"**/*.nut.ts\" --slow 4500 --timeout 600000 --parallel",
+    "test:nuts:snapshots": "nyc mocha \"test/nuts/snapshots.nut.ts\" --slow 4500 --timeout 600000 --parallel",
```

Comment:
This is not being used (`test:nuts` already picks every *.nut.ts file), do you plan to break them up like plugin-source NUTs?

Reply:
I like to be able to run them more granularly, even if just at dev time.  The other thing is that while the org:shape need to go separately by OS because of collisions, these really don't.

TBD, I guess